backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt

```diff
@@ -136,7 +136,7 @@ open class ReleasedDataModel(
             mapOf(
                 ("accession" to TextNode(rawProcessedData.accession)),
                 ("version" to LongNode(rawProcessedData.version)),
-                (HEADER_TO_CONNECT_METADATA_AND_SEQUENCES to TextNode(rawProcessedData.submissionId)),
+                ("submissionId" to TextNode(rawProcessedData.submissionId)),
```

Comment:
Why don't we use `HEADER_TO_CONNECT_METADATA_AND_SEQUENCES_ALTERNATE_FOR_BACKCOMPAT` here?